liquidctl/driver/hydro_platinum.py

```diff
@@ -9,6 +9,7 @@
 - Corsair Hydro H100i Pro XT
 - Corsair Hydro H115i Pro XT
 - Corsair Hydro H150i Pro XT
+- Corsair iCUE H100i RGB ELITE
```

Comment:
To avoid too much marketing-case, and also to help users spot the part that distinguishes this from other "elite" models, let's call it:

```suggestion
- Corsair iCUE H100i Elite RGB
```